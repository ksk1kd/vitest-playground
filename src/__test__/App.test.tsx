import { render } from '@testing-library/react'
import { describe, expect, it } from 'vitest'
import App from '../App'

describe('App', () => {
  it('should render correctly', () => {
    const { container } = render(<App />)
    expect(container).toMatchSnapshot()
  })
})
