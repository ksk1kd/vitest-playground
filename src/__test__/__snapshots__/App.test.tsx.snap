// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > should render correctly 1`] = `
<div>
  <div
    class="container"
  >
    Contents
  </div>
</div>
`;
